d2bs/kolbot/libs/common/Misc.js

```diff
@@ -2344,7 +2354,17 @@ CursorLoop:
 		if (value > 0) {
 			getPacket(1, 0x1d, 1, stat, 1, value);
 		}
-	}
+	},
+
+	addListener: function (packetType, callbackFunc) { // more specialized wrapper for addEventListener
+		if (typeof packetType == 'number')
+			packetType = [packetType];
+		if (typeof packetType == 'object' && packetType.length)
+			return addEventListener('gamepacket', packet=>(packetType.indexOf(packet[0]) > -1 ? callbackFunc(packet) : false));
+		return null;
+	},
+
+	removeListener: removeEventListener, // just a wrapper
```

Comment:
This should not require same args as removeEventListener since it is explicitly for gamepacket.